accumulo/src/main/java/org/vertexium/accumulo/util/ConnectorUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.vertexium.accumulo.util;
+
+import org.apache.accumulo.core.client.Connector;
+import org.apache.accumulo.core.client.TableNotFoundException;
+import org.apache.accumulo.core.client.impl.ClientContext;
+import org.apache.accumulo.core.client.impl.ConnectorImpl;
+import org.vertexium.VertexiumException;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+public class ConnectorUtils {
+    public static ClientContext getContext(Connector connector) {
+        if (!(connector instanceof ConnectorImpl)) {
+            throw new VertexiumException("Invalid connector type. Expected " + ConnectorImpl.class.getName() + " found " + connector.getClass().getName());
+        }
+        try {
+            Field f = ConnectorImpl.class.getDeclaredField("context");
```

Comment:
This won't be necessary when we upgrade to Accumulo 2.0. They seem to have added a get method: https://github.com/apache/accumulo/blob/master/core/src/main/java/org/apache/accumulo/core/clientImpl/ConnectorImpl.java#L74

Reply:
Excellent